great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -307,7 +312,11 @@ def get_batch_data_and_markers(
 
     def _apply_splitting_and_sampling_methods(self, batch_spec, batch_data):
         if batch_spec.get("splitter_method"):
-            splitter_fn = getattr(self, batch_spec.get("splitter_method"))
+            splitter_method_name: str = batch_spec.get("splitter_method")
```

Comment:
Nitpick - Could we use the following pattern?

```
splitter_method_name: Optional[str] = batch_spec.get("splitter_methid")
if splitter_method_name:
    ...
```

Just to prevent having to retrieve twice.